### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by Odi-3** [syphon-org/syphon#792] [General]
I find this project very interesting, but I don't understand if it is still active, given that the last update dates back to October 3, 2022.
Thank you for your feedback.

**Comment by dnisbetjones** [comment]
Thanks very much for your question. Unfortunately, development on Syphon is currently inactive due to time constraints. I don't have any sort of ETA I can give as to when development will pick up again.

**Reply by Odi-3** [reply]
I am sorry about that, but I very much hope that it will resume soon. In the meantime I'll be happy to contribute as much as I can, with some translations and bug reports.
Thank you for your work.

**Reply by marmonto** [reply]
I came here to ask the same thing. I'm expecting that you can re-take the project.
I'll be waiting for more work in translation. 
Thanks!

**Comment by EdGeraghty** [comment]
See #794
